libraries/Microsoft.Bot.Builder.Prompts/Choices/ListStyle.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
```

Comment:
Thanks for moving this to it's own class. Having 1 class per file is where we need to be for everything.

Reply:
I think it's the way it should always be.
If you want to, when I have some time, I could go through all the files, first in this project, later on the rest, and do the same thing like this.
But, I would like one of you to "approve" this before I start to.